instrumentation/apache-camel-2.20/javaagent/src/test/groovy/test/SqsCamelTest.groovy

```diff
@@ -33,6 +32,9 @@ class SqsCamelTest extends AgentTestRunner {
 
   def setupSpec() {
 
+    /**
+     * Temporarily using emq instead of localstack till the latter supports AWS trace propagation
```

Comment:
Do you mind filing / linking an issue? https://github.com/localstack/localstack/issues

Reply:
Done: https://github.com/localstack/localstack/issues/3528